unlock-app/src/__tests__/components/creator/CreatorLockForm.test.js

```diff
@@ -99,9 +93,9 @@ describe('CreatorLockForm', () => {
       }
     })
     it('key expiration is a negative number', () => {
-      const wrapper = makeLockForm({ expirationDuration: -1 })
+      const wrapper = makeLockForm({ expirationDuration: -2 * 86400 })
```

Comment:
Nit: when using numeric values like this, feel free to put the corresponding "human" format like "2 days" as comments. This makes it easier to skip the math ;)

Reply:
Just for the sake of argument, when I do things like this I like to have a named variable like `secondsInADay` so that you end up with `2 * secondsInADay` which is a bit more self-documenting and resilient against change (e.g., if we change it in the future to being 3 days, there won't be a comment that can be out of sync with the code).

Reply:
I like that @cnasc thanks